src/main/webapp/scripts/controllers/AffiliateManagementController.js

```diff
@@ -71,11 +75,14 @@ mldsApp.controller('AffiliateManagementController', [
                     if (_.size($scope.affiliates) > 0) {
                         $scope.noResults = false;
                     }
-                    //$log.log("...number of query results=" + response.data.totalResults);
-                    //$log.log("...total pages=" + response.data.totalPages);
                     $scope.page = $scope.page + 1;
                     $scope.downloadingAffiliates = false;
                     $scope.totalResults=response.data.totalResults;
+                    $scope.totalPages=response.data.totalPages;
+                    if ($scope.affiliates.length >= $scope.totalResults) {
+                        $log.log("all retrieved true" );
```

Comment:
If you made this "All available affiliates retrieved" then it would be more meaningful.

Reply:
Yes, changed it